doc/manual/installation/upgrading.xml

```diff
@@ -7,15 +7,21 @@
   <title>Upgrading Nix</title>
 
   <para>
-    Multi-user Nix users on macOS can upgrade Nix by running
+    Multi-user Nix users on macOS can upgrade Nix by running this:
     <command>sudo -i sh -c 'nix-channel --update &amp;&amp; nix-env
-    -iA nixpkgs.nix'; sudo launchctl stop org.nixos.nix-daemon; sudo
-    launchctl start org.nixos.nix-daemon</command>.
+      -iA nixpkgs.nix'; sudo launchctl stop org.nixos.nix-daemon; sudo
+      launchctl start org.nixos.nix-daemon</command>
```

Comment:
Does this actually reload the service and start the new version? I think this might explain an upgrade issue I noticed yesterday.

Reply:
Yes, that's what it's supposed to do. I've heard of two issues:
1) If you have a package installed that depends on `nix` version 1.x, the updated version 2.x might not take over.
2) For a multi-user installation, you shouldn't install `nix` as the current user.